packages/design-system/src/components/SelectField/SelectField.md

```diff
@@ -49,6 +49,7 @@ const getSelectedItemBody = props => {
     getSelectedItemBody={getSelectedItemBody}
     selected={state.selectedItem}
     searchPlaceholder='Search...'
+    selectHeader="Select from items below"
```

Comment:
https://take.ms/ZeJzK - should the scroll on keypress behave like that?

Reply:
I think so, you cannot select the header. What do you propose?

Reply:
I tested it one more time and this scrolling with the keyboard doesn't work very well so let's leave it now
I expected that when the first element is selected it will be scrolled to the top and I will be able to see the header